packages/components/src/input-control/input-base.tsx

```diff
@@ -73,7 +73,7 @@ export function InputBase(
 	const hideLabel = hideLabelFromVision || ! label;
 
 	return (
-		// @ts-expect-error The `direction` prop from Flex (FlexDirection) conflicts with legacy SVGAttributes `direction` (string) that come from React intrinsic prop definitions
+		// @ts-expect-error The `direction` prop from Flex (FlexDirection) conflicts with legacy SVGAttributes `direction` (string) that come from React intrinsic prop definitions.
```

Comment:
This raises a flag, but I guess it's only an explanation for the expected TypeScript error.